lib/aggregate/attribute/date_time.rb

```diff
@@ -1,12 +1,22 @@
 # frozen_string_literal: true
 
 class Aggregate::Attribute::DateTime < Aggregate::Attribute::Builtin
+  def self.available_options
+    super + [:format]
+  end
+
   def load(value)
     ActiveSupport::TimeZone['UTC'].parse(value).in_time_zone
   end
 
   def store(value)
-    value.utc.rfc822
+    if @options.dig(:aggregate_db_storage_type) == :elasticsearch
+      value.utc.iso8601
+    elsif @options[:format]
```

Comment:
Would it make sense to allow for `:format` to override the default value assigned by `:aggregate_db_storage_type` being `:elasticsearch`? Is there a time where we'd provide both and not want to use `:format`?